sagemaker-remote-function/pytorch_mnist_sample_notebook/pytorch_mnist.ipynb

```diff
@@ -572,11 +574,147 @@
    "outputs": [],
    "source": [
     "with Run(\n",
-    "    experiment_name=\"pytorch-mnist-decorator\", run_name=\"run-1\", sagemaker_session=sm_session\n",
+    "    experiment_name=\"pytorch-mnist-decorator\", run_name=f\"run-{uuid.uuid4()}\", sagemaker_session=sm_session\n",
     ") as run:\n",
     "    trained_model = perform_train(train_set, test_set)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Excluding current files/folder\n",
```

Comment:
It is not a good place to show this capability here. 

    "        IncludeLocalWorkDir: true\n",
    "        CustomFileFilter:\n",

These two fields are mainly for users develop in a modular python package, and they need to package up the workspace and send to jobs. 

See my comments below.